lib/globus/client/identity.rb

```diff
@@ -39,7 +39,7 @@ def extract_id(data)
         # Select identity with "used" or "private" status
         matching_users = identities.select { |id| id["username"] == @email }
         active_users = matching_users.select { |user| (user["status"] == "used" || user["status"] == "private") }
-        raise StandardError "No matching active Globus user found for #{@email}." if active_users.empty?
```

Comment:
This needed a comma between the exception class and the message. It's not exactly a syntax/parser issue and the test as written before hid this bug since `NoMethodError` is a subclass of `StandardError`.